python/ray/air/_internal/tensorflow_utils.py

```diff
@@ -98,7 +102,9 @@ def get_dtype(dtype: Union[np.dtype, pa.DataType]) -> tf.dtypes.DType:
 
     def get_shape(dtype: Union[np.dtype, pa.DataType]) -> Tuple[int, ...]:
         shape = (None,)
-        if isinstance(dtype, ArrowTensorType):
+        if isinstance(
+            dtype, (ArrowTensorType, ArrowTensorTypeV2, ArrowVariableShapedTensorType)
```

Comment:
nit, let's add a util function `is_arrow_tensor_extension_type`, as it's duplicated many times

Reply:
Good call